library/psa_crypto.c

```diff
@@ -453,7 +453,7 @@ static psa_status_t validate_unstructured_key_bit_size( psa_key_type_t type,
     switch( type )
     {
         case PSA_KEY_TYPE_RAW_DATA:
-#if defined(MBEDTLS_MD_C)
+#if defined(MBEDTLS_PSA_BUILTIN_ALG_HMAC)
```

Comment:
Rather remove it as I think we need this when HMAC is provided by an accelerator.